walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/constraints/VcPathRegexConstraint.kt

```diff
@@ -0,0 +1,55 @@
+package com.microsoft.walletlibrary.requests.requirements.constraints
+
+import com.jayway.jsonpath.Configuration
+import com.jayway.jsonpath.JsonPath
+import com.jayway.jsonpath.Option
+import com.microsoft.walletlibrary.did.sdk.credential.models.VerifiableCredentialContent
+import com.microsoft.walletlibrary.did.sdk.util.log.SdkLog
+import com.microsoft.walletlibrary.util.NoMatchForVcPathRegexConstraintException
+import com.microsoft.walletlibrary.verifiedid.VerifiableCredential
+import com.microsoft.walletlibrary.verifiedid.VerifiedId
+import kotlinx.serialization.json.Json
+import java.util.regex.Pattern
+
+class VcPathRegexConstraint(
+    internal val path: List<String>, internal val pattern: String
+) : VerifiedIdConstraint {
+    override fun doesMatch(verifiedId: VerifiedId): Boolean {
```

Comment:
is this internal?

Reply:
Methods inherit the visibility from interface and it cannot be made internal in interface. Since the classes are internal, it should be okay.